peregrine/analysis/print_res.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```

Comment:
Sure, I will change the file name.

About the data format. Peregrine uses Python pickle format for acquisition and tracking results. It is true though that tracking_loop.py generates the track results in CSV format at the moment. The CSV output is used by print_res.py script, which plots a requested parameter. The CSV format was chosen because it is human readable and presents data in a table form. I sometimes find it useful to look at the tracking results in a raw numerical form. However, I think, that print_res.py can be redone to read tracking data presented in Python pickle format and another script would take the tracking results in the Python pickle format and create CSV output out of it. This would be sort of a Unix way of doing things: one tool for one task. At the same time the main standard format for any result data in Peregrine would remain Python pickle as it was designed earlier.